lib/shortcode.php

```diff
@@ -83,7 +83,7 @@ function anvato_shortcode_get_parameters( $attr ) {
 		}
 	}
 
-	if ( !empty( $mcp['mcp']['tkx_key'] ) ) {
+	if ( !empty( $mcp['mcp']['tkx_key'] ?? '' ) ) {
```

Comment:
Just for educational purposes I would like to know why is needed to do this inside `empty` function, since it returns true when the value is null or undefined

Reply:
If `$mcp` is `null` (which can happen) then this will throw a warning about trying to access offset `mcp` of `null`. Throwing a warning in this situation (rather than ignoring it or throwing a notice) is new behavior in PHP 8. _Functionally_ nothing changes here; this PR is because https://nexstardigital.atlassian.net/browse/WC-1518 requested to clear out warnings.

Reply:
got it, thanks